.values.cjs

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const testSuffixBenchmark = "bench";
+const testSuffixTest = "test";
+
+const testTypeTests = "tests";
+const testTypeBenchmark = "benchmark";
+const testTypeIntegration = "integration";
+const testTypeUnit = "unit";
+
+const compiledDir = "{build,lib}";
+const dependenciesDir = "node_modules";
+const githubDir = ".github";
+const mocksDir = "__mocks__";
+const packagesDir = "packages";
+const scriptsDir = "scripts";
+const srcDir = "src";
+const reportsDir = "_reports";
+const tempDir = ".temp";
+const testDataDir = "testdata";
+const testsDir = "__tests__";
+
+let packagesExpr = "*";
+let packagesList = [packagesExpr];
+if (process.env.TEST_PACKAGES !== undefined) {
+  packagesExpr = process.env.TEST_PACKAGES;
+  packagesList = process.env.TEST_PACKAGES.split(",");
+  if (packagesList.length > 1) {
+    packagesExpr = `{${packagesExpr}}`;
+  }
+}
+
+module.exports = {
+  // Directories
```

Comment:
It may be more intuitive to expose these directoreis through [the `paths.js` file](https://github.com/ericcornelissen/webmangler/blob/8680b73fbd7a36850039be49f8aaa464545a9a35/scripts/paths.js) (e.g. it already has a [`packagesDir`](https://github.com/ericcornelissen/webmangler/blob/8680b73fbd7a36850039be49f8aaa464545a9a35/scripts/paths.js#L15)).